python_benchmarks/global_params.py

```diff
@@ -12,8 +13,8 @@ def generate_input_sizes(dims: Union[int, List] = 2) -> List[Tuple]:
     # TODO: Add more input sizes.
     for dim in dims:
         if dim == 2:
-            range_outer = [2**i for i in range(1, 5)]
-            range_inner = [32 * 1024 * 2**i for i in range(11)]
+            range_outer = [2**i for i in range(4, 9)]  # {16, 256}
```

Comment:
All these range changes are due to the validation failures?

Reply:
Yes, in `layernorm`.

Reply:
OK, please leave a TODO comment as we are planning to address it